Client/Frontend/Home/JumpBackIn/SyncedTabCell.swift

```diff
@@ -0,0 +1,333 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import UIKit
+
+struct FxHomeSyncedTabCellViewModel {
+    let titleText: String
+    let descriptionText: String
+    let url: URL
+    let tag: Int
+    var syncedDeviceImage: UIImage?
+    var heroImage: UIImage?
+    var accessibilityLabel: String {
+        return "\(cardTitleText): \(titleText), \(descriptionText)"
+    }
+    var cardTitleText: String {
+        return .FirefoxHomepage.JumpBackIn.SyncedTabTitle
+    }
+    var syncedTabsButtonText: String {
+        return .FirefoxHomepage.JumpBackIn.SyncedTabShowAllButtonTitle
+    }
+}
+
+/// A cell used in FxHomeScreen's Jump Back In section
+class SyncedTabCell: UICollectionViewCell, ReusableCell {
+
+    struct UX {
+        static let generalCornerRadius: CGFloat = 12
+        static let stackViewShadowRadius: CGFloat = 4
+        static let stackViewShadowOffset: CGFloat = 2
+        static let heroImageSize =  CGSize(width: 108, height: 80)
+        static let fallbackFaviconSize = CGSize(width: 56, height: 56)
+        static let syncedDeviceImageSize = CGSize(width: 24, height: 24)
+        static let itemTitleTopAnchorConstant: CGFloat = 64
+        static let itemTitleTopAnchorCompactPhoneConstant: CGFloat = 24
+    }
+
+    private var syncedDeviceIconCenterConstraint: NSLayoutConstraint?
+    private var syncedDeviceIconFirstBaselineConstraint: NSLayoutConstraint?
+    private var itemTitleTopConstraint: NSLayoutConstraint!
+    private var showAllSyncedTabsAction: ((UIButton) -> Void)?
+    private var openSyncedTabAction: (() -> Void)?
+
+    // MARK: - UI Elements
+    private let cardTitle: UILabel = .build { label in
+        label.adjustsFontForContentSizeCategory = true
+        label.font = DynamicFontHelper.defaultHelper.preferredFont(withTextStyle: .headline, size: 16)
+        label.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.cardTitle
+    }
+
+    private let syncedTabsButton: UIButton = .build { button in
+        button.titleLabel?.font = DynamicFontHelper().preferredFont(withTextStyle: .subheadline, size: 12)
+        button.titleLabel?.adjustsFontForContentSizeCategory = true
+        button.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.showAllButton
+    }
+
+    let heroImage: UIImageView = .build { imageView in
+        imageView.contentMode = .scaleAspectFill
+        imageView.clipsToBounds = true
+        imageView.layer.masksToBounds = true
+        imageView.layer.cornerRadius = UX.generalCornerRadius
+        imageView.backgroundColor = .clear
+        imageView.accessibilityIdentifier = AccessibilityIdentifiers.FirefoxHomepage.SyncedTab.heroImage
```

Comment:
Since UIImageView isAccessibilityElement is set to `false` by default, is it still worth it to set an `accessibilityIdentifier`? This won't be accessible through UI tests no?

Reply:
I was speaking to a QA in the past and they said that depending on the software you use for UI Testing you can find the element if you have the accessibilityIdentifier.

Reply:
Oh ok! Doesn't hurt then I guess